### DISCUSSION TITLE
post procces in phaser extension support

### DISCUSSION THREAD

**Discussion by Trusiak** [enable3d/enable3d#85] [Help / Questions] [answered]
Hello!
does new version of enable3d will have phaser3d extension post proccess support, or only simple enable3d will have post processing?

and what about release date??

**Comment by yandeu** [comment]
The phaser extension already supports post-processing.

**Comment by Trusiak** [comment]
and what about examples "how to" use it on phaser extension?

**Comment by yandeu** [comment, marked as answer]
It works in the same way as [this example](https://enable3d.io/examples/post-processing.html) but with a `this.third.` in front.

```t
this.third.composer()
this.third.renderer()
this.third.scene()
this.third.camera()
// and so on
```

_Post-processing has only effect on 3d elements_
